base_geoengine/static/src/js/views/geoengine/geoengine_controller/geoengine_controller.esm.js

```diff
@@ -40,24 +44,85 @@ export class GeoengineController extends Component {
                 offset: offset,
                 limit: limit,
                 total: count,
-                onUpdate: async ({offset, limit}) => {
```

Comment:
There might be a mistake here, the parameter names must be the same that odoo is using: offset and limit. If they don't match the pager can't be updated accordingly.
I could swear this was working when I checked it, has it been changed recently?

Reply:
Fixed, thanks!